### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/core/basesyntax/FileWork.java [hunk 1]
```diff
@@ -8,7 +15,29 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private static final String WORD_STARTS_WITH = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        Path inputFilePath = Paths.get(fileName);
+        ArrayList<String> inputFile = new ArrayList<>();
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:22]: in this case a good practice is to use type of a parent
```suggestion
        List<String> inputFile = new ArrayList<>();
```

File: src/main/java/core/basesyntax/FileWork.java [hunk 2]
```diff
@@ -8,7 +15,29 @@
...
+        try {
+            inputFile = (ArrayList<String>) Files.readAllLines(inputFilePath);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:24]: we do not need casting here

File: src/main/java/core/basesyntax/FileWork.java [hunk 3]
```diff
@@ -8,7 +15,29 @@
...
+        } catch (IOException e) {
+            System.out.println("File does not exist");
+        }
+        ArrayList<String> result = new ArrayList<>();
+        for (String string : inputFile) {
+            String[] words = string.split(" ");
+
+            for (String word : words) {
+                if (word.replaceAll("\\W", "").toLowerCase().startsWith(WORD_STARTS_WITH)) {
+                    result.add(word.replaceAll("\\W", "").toLowerCase());
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:34]: let's first modify the `word` String, and then check if it starts with `WORD_STARTS_WITH`

File: src/main/java/core/basesyntax/FileWork.java [hunk 4]
```diff
@@ -8,7 +15,29 @@
...
+                }
+            }
+        }
+        Collections.sort(result);
+
+        return result.toArray((new String[result.size()]));
+
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/FileWork.java:41]: you can remove this blank line

File: src/main/java/core/basesyntax/FileWork.java (hunk 1 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/FileWork.java:22]: fixed

File: src/main/java/core/basesyntax/FileWork.java (hunk 2 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/FileWork.java:24]: fixed

File: src/main/java/core/basesyntax/FileWork.java (hunk 3 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/FileWork.java:34]: done

File: src/main/java/core/basesyntax/FileWork.java (hunk 4 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/FileWork.java:41]: removed

**Pull request closed without merging**